static/js/chart/draw.ts

```diff
@@ -674,18 +679,20 @@ function buildInChartLegend(
         .attr("x2", `${LEGEND.dashWidth - 3}`)
         // Default color to be black here.
         .attr("stroke", "#000000")
-        .attr("stroke-dasharray", `${plotParams.dashes[placeName]}`);
-      // Draw the text.
-      lgGroup
-        .append("text")
-        .attr("transform", `translate(${LEGEND.dashWidth}, 0)`)
-        .attr("y", "0.3em")
-        .attr("dy", "0em")
-        .style("font-size", "14")
-        .text(placeName)
-        .call(wrap, legendTextdWidth);
-      yOffset += lgGroup.node().getBBox().height + LEGEND.lineMargin;
+        .attr("stroke-dasharray", `${plotParams[place[1]].dash}`);
+      dashWidth = LEGEND.dashWidth;
     }
+    // Draw the text.
+    lgGroup
+      .append("text")
+      .attr("transform", `translate(${dashWidth}, 0)`)
+      .attr("y", "0.3em")
+      .attr("dy", "0em")
+      .style("font-size", "14")
```

Comment:
svg styles such as font-size, stroke, etc should be in the css as much as possible

Reply:
Moved non svg style to scss